expander.c

```diff
@@ -281,14 +331,36 @@ bgpq_expanded_macro_limit(char *as, struct bgpq_expander *b,
 		    req->depth + 1 < b->maxdepth)) {
 			bgpq_expander_add_already(b, as);
 			if (pipelining) {
+				if (b->usesource) {
+					char *source = bgpq_get_source(as);
```

Comment:
Be careful not to obfuscate the code by initializing variables in the declarations.

Reply:
I'm not following you? This style is all throughout the code: https://github.com/bgp/bgpq4/blob/016da02b9ebb2f103738109caf9f406374efd914/expander.c#L186

Reply:
I suggest to move `char *source` to the top of the function.

I'm aware the code still has a long way to go to comply with OpenBSD's [style](http://man.openbsd.org/style) guide.